ProcessMaker/screen-builder
# observation/FOUR-17519 Added a isVisible validation when the input is not visible in mobile

## Issue & Reproduction Steps
`Loading submit button` does not work in mobile when `show for mobile` is enabled

Expected behavior: 
In mobile version the loading label should be displayed and then the next task should be displayed

Actual behavior: 
In mobile version the loading label is displayed but the next task is not displayed.

## Solution
Added a isVisible validation when the input is not visible in mobile

## How to Test
Go to server
Create a simple process (start event - task - task - end event)
Create a screen
Create a form_input
check the Make Required option
Unchecked the show for desktop and show for mobile options
Create a submit button
Check loading Submit Button option
Add a text in the loading label field
Run a Case
Press the submit button

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-17519

ci:next
ci:vue-form-elements:next
